### PR TITLE
Minor updates to SparkR vignette

### PR DESCRIPTION
**Pull request by shivaram**

### CODE REVIEW

File: R/pkg/vignettes/sparkr-vignettes.Rmd [hunk 1]
```diff
@@ -15,40 +15,35 @@ SparkR is an R package that provides a light-weight frontend to use Apache Spark
...
+We begin with an example running on the local machine and provide an overview of the use of SparkR: data ingestion, data processing and machine learning.
 
 First, let's load and attach the package.
 ```{r, message=FALSE}
 library(SparkR)
 ```
 
 
-To use SparkR, you need an Apache Spark package where backend codes to be called are compiled and packaged. 
+To use SparkR, you need an Apache Spark installation which will execute SparkR programs.
```
Comment (junyangq) [inline R/pkg/vignettes/sparkr-vignettes.Rmd]: This part is meant to explain why we still need Spark even we already have SparkR. I'm not sure if this is enough for them to totally understand?

File: R/pkg/vignettes/sparkr-vignettes.Rmd [hunk 2]
```diff
@@ -125,14 +120,14 @@ paste("Spark", packageVersion("SparkR"))
 ```
 It should be used both on the local computer and on the remote cluster.
 
-To connect, pass the URL of the master node to `sparkR.session`. A complete list can be seen in [Spark Master URLs](http://spark.apache.org/docs/latest/submitting-applications.html#master-urls). 
-For example, to connect to a local cluster master, we can call
+To connect, pass the URL of the master node to `sparkR.session`. A complete list can be seen in [Spark Master URLs](http://spark.apache.org/docs/latest/submitting-applications.html#master-urls).
+For example, to connect to a standalone Spark master, we can call
```
Comment (junyangq) [inline R/pkg/vignettes/sparkr-vignettes.Rmd]: a local standalone Spark master?

**Comment by junyangq** [comment]
Thanks @shivaram for the PR. The updates look good to me, with just a few comments above.

File: R/pkg/vignettes/sparkr-vignettes.Rmd (hunk 1 above)
Comment (shivaram) [reply R/pkg/vignettes/sparkr-vignettes.Rmd]: Yeah - I was thinking that we don't want to confuse them too much about SparkR vs. Spark etc. I added back a line about why they need the Spark installation

**Comment by shivaram** [comment]
@junyangq Will be great if you can do one more pass and open a PR against apache/spark

One more thing is that we should write a script to automate building the vignette and putting it in the installation. Do you know what are the steps for that ?

**Comment by junyangq** [comment]
Thanks. Yeah I think I can take care of the script. Do you want to put it together in `create-doc.sh` or a separate one?

**Comment by shivaram** [comment]
Adding it to `create-doc.sh` should be fine.

**Pull request merged**